assets/queries/openAPI/string_schema_without_maximum_length/query.rego

```diff
@@ -7,14 +7,14 @@ CxPolicy[result] {
 	openapi_lib.check_openapi(doc) != "undefined"
 
 	[path, value] := walk(doc)
-	schema = value.schema
+	schema_kind = openapi_lib.get_schema(value)
 	info := openapi_lib.is_operation(path)
 	openapi_lib.content_allowed(info.operation, info.code)
-	openapi_lib.undefined_properties_in_schema(doc, schema, "string", "maxLength")
+	openapi_lib.undefined_properties_in_schema(doc, value, schema_kind, "string", "maxLength")
```

Comment:
I think single rule example on `assets/queries/openAPI/numeric_schema_without_format/query.rego` applies here too, but using `maxLength` instead `format` and instead `is_numeric_type`, use `value.type == "string"`